src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/attachment.yml

```diff
@@ -0,0 +1,66 @@
+admin_attachments_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Attachment:index
+    _legacy_controller: AdminAttachments
+    _legacy_link: AdminAttachments
```

Comment:
While migrating, you should comment `_legacy_link` out, this will allow you to access both legacy & new pages. This applies for other routes as well, you should uncomment it when page is fully migrated and legacy controller is deleted.